bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/InstarHandler.java

```diff
@@ -62,43 +63,61 @@ public void channelRead(@Nullable ChannelHandlerContext ctx, @Nullable Object ms
             ipCameraHandler.logger.trace("HTTP Result back from camera is \t:{}:", content);
             switch (requestUrl) {
                 case "/param.cgi?cmd=getinfrared":
-                    if (content.contains("var infraredstat=\"auto")) {
+                    if (content.contains("var infraredstat=\"auto") || content.contains("infraredstat=\"2")) {
                         ipCameraHandler.setChannelState(CHANNEL_AUTO_LED, OnOffType.ON);
                     } else {
                         ipCameraHandler.setChannelState(CHANNEL_AUTO_LED, OnOffType.OFF);
                     }
                     break;
                 case "/param.cgi?cmd=getoverlayattr&-region=1":// Text Overlays
-                    if (content.contains("var show_1=\"0\"")) {
+                    if (content.contains("var show_1=\"0\"") || content.contains("show=\"0\"")) {
                         ipCameraHandler.setChannelState(CHANNEL_TEXT_OVERLAY, StringType.EMPTY);
                     } else {
                         value1 = Helper.searchString(content, "var name_1=\"");
                         if (!value1.isEmpty()) {
                             ipCameraHandler.setChannelState(CHANNEL_TEXT_OVERLAY, StringType.valueOf(value1));
+                        } else {
+                            value1 = Helper.searchString(content, "name=\"");
+                            if (!value1.isEmpty()) {
+                                ipCameraHandler.setChannelState(CHANNEL_TEXT_OVERLAY, StringType.valueOf(value1));
+                            }
                         }
                     }
                     break;
-                case "/cgi-bin/hi3510/param.cgi?cmd=getmdattr":// Motion Alarm
-                    // Motion Alarm
+                case "/cgi-bin/hi3510/param.cgi?cmd=getmdattr":// Motion Alarm old
                     if (content.contains("var m1_enable=\"1\"")) {
                         ipCameraHandler.setChannelState(CHANNEL_ENABLE_MOTION_ALARM, OnOffType.ON);
                     } else {
                         ipCameraHandler.setChannelState(CHANNEL_ENABLE_MOTION_ALARM, OnOffType.OFF);
                     }
                     break;
-                case "/cgi-bin/hi3510/param.cgi?cmd=getaudioalarmattr":// Audio Alarm
-                    if (content.contains("var aa_enable=\"1\"")) {
+                case "/param.cgi?cmd=getalarmattr":// Motion Alarm new
+                    if (content.contains("armed=\"1")) {
```

Comment:
Isn't a final `\"` missing to match value 1?

Reply:
Agree safer in case api gets changed. Made change so vale 11, 111 etc do not match